querybook/server/logic/datadoc.py

```diff
@@ -813,6 +813,15 @@ def get_data_doc_editor_by_id(id, session=None):
 def get_data_doc_editors_by_doc_id(data_doc_id, session=None):
     return session.query(DataDocEditor).filter_by(data_doc_id=data_doc_id).all()
 
+# Return Instance of unique data
```

Comment:
use docstring `"""..."""` instead, or just remove it as this function is also quite self-explanatory